crac/src/main/java/io/micronaut/crac/resources/redis/RedisClientResource.java

```diff
@@ -32,14 +32,18 @@
 /**
  * Destroys any RedisClient beans before checkpointing.
  *
+ * @deprecated Since 1.2.3, this is covered by {@link RedisNamedConfigResource} now, which can handle multiple named configurations
  * @author Tim Yates
  * @since 1.2.1
  */
+@Deprecated
 @Experimental
 @EachBean(RedisClient.class)
 @Requires(classes = {RedisClient.class})
 @Requires(bean = CracResourceRegistrar.class)
 @Requires(property = RedisClientResource.ENABLED_PROPERTY, defaultValue = StringUtils.TRUE, value = StringUtils.TRUE)
+// Disable this resource in a binary compat way
+@Requires(missing = RedisCacheResource.class)
```

Comment:
if this class is unecessary delete it. It was maked as `@Experimental`

Reply:
Done in https://github.com/micronaut-projects/micronaut-crac/pull/92/commits/f0bdfd54eff02ebb04d5cc33f2fb881215c0a8ae